client/extensions/woocommerce/woocommerce-services/lib/initialize-labels-state/index.js

```diff
@@ -42,6 +42,35 @@ export default data => {
 	const hasOriginAddress = addressFilled( formData.origin );
 	const hasDestinationAddress = addressFilled( formData.destination );
 
+	const customsItemsData = {};
+	forEach( formData.selected_packages, ( { items } ) => {
+		items.map(
+			( {
+				product_id,
+				weight,
+				name,
+				attributes,
+				description,
+				value,
+				hs_tariff_number,
+				origin_country,
+				parent_product_id,
+			} ) => {
+				const attributesStr = attributes ? ' (' + attributes + ')' : '';
+				const defaultDescription = name.substring( name.indexOf( '-' ) + 1 ).trim() + attributesStr;
+				customsItemsData[ product_id ] = {
+					defaultDescription,
+					description: description || defaultDescription,
+					weight,
+					value,
+					tariffNumber: hs_tariff_number || '',
+					originCountry: origin_country || formData.origin.country,
+					parentProductId: parent_product_id,
```

Comment:
What is the `parent_product_id` used for?

Reply:
See [this comment](https://github.com/Automattic/wp-calypso/pull/24974#discussion_r203815455) for more context. `parent_product_id` was used a few iterations ago to render a link to the `Edit product` page, but that link was scraped from the UI, so now it's not used at all.

Do you think we should keep it just in case, or remove it from here and the plugin endpoint? I'm not a fan of dead code but I could see us wanting that link in the near future.

Reply:
Happy to keep for the link 👍.

Reply:
I just realized I removed `parent_product_id` from the plugin endpoint a while back. Jeff asked me the same question as you :)
So now there's no point in keeping it here. I've removed it in https://github.com/Automattic/wp-calypso/pull/24974/commits/44892d3f83b6124a7bca236344b8a61903ef5921